developers/index.html

```diff
@@ -27,8 +27,8 @@ <h2>SQL References</h2>
 <ul>
     <li><a href="https://dev.mysql.com/doc/">MySQL</a></li>
     <li><a href="http://www.postgresql.org/docs/">PostgreSQL</a></li>
-    <li><a href="http://www.oracle.com/technology/documentation/database.html">Oracle</a></li>
-    <li><a href="http://savage.net.au/SQL/">ANSI SQL</a>
```

Comment:
Website moved here, I think?
https://ronsavage.github.io/SQL/

Reply:
http://savage.net.au/SQL links to https://github.com/ronsavage/SQL but the `.io` version renders the docs up real nice so taht does sound better